### ISSUE TITLE
Warning reported when used with Webpack 

### ISSUE THREAD

**Issue by leowmjw** [GuillaumeLeclerc/vue-google-maps#14]
Using vue-google-maps with the standard webpack setup generated by the vue-cli.  When a change is made; it hot reloads but shows the following warning.

```
You have included the Google Maps API multiple times on this page. This may cause unexpected errors.
```

Any workaround?  Thanks!

**Comment by GuillaumeLeclerc** [comment]
are you sure you are not adding it yourself. `vue-google-maps` already loads the google library for you. If you do it already (with `<script>`) you will get an error

**Comment by GuillaumeLeclerc** [comment]
Any news about this ?

**Comment by leowmjw** [comment]
@GuillaumeLeclerc You are right; I did have the api already loaded with <script>.  Thanks for the clarification.

**Comment by lesion** [comment]
I'm having this issue using webpack and hot-reload.
I've spent some time to check out what's going on and found that in manager.js 
there's a setUp [check](https://github.com/GuillaumeLeclerc/vue-google-maps/blob/master/src/manager.js#L43) but I did not found where `setUp` is assigned to true !
Putting a `setUp = true` after the check and removing the throwing exception works ok (throwing 
exception inside a component makes hot-reload reload the entire page).
I'm not PR 'cause really don't know if there's something else needed to check here (probably some
reinitialization? do we need a [beforeDestroy](http://vuejs.org/guide/instance.html#Lifecycle-Diagram) inside map.vue component?)

PS. changes here => 6ffc11978687f0ffe353a44dc2640f268841ff87
